credit_limit_sale_order_approval/i18n/credit_limit_sale_order_approval.pot

```diff
@@ -0,0 +1,86 @@
+# Translation of Odoo Server.
```

Comment:
There's no need now of the pot file with Transifex process. Remove it for clarity.